examples/units/bar_demo2.py

```diff
@@ -3,12 +3,12 @@
 Bar demo with units
 ===================
 
-plot using a variety of cm vs inches conversions.  The example shows
-how default unit instrospection works (ax1), how various keywords can
+A plot using a variety of centimetre vs inches conversions.  This example shows
```

Comment:
vs inches -> and inch

Is this line not too long? Or is everything else just wrapped a bit short?